mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -44,6 +44,45 @@
 using namespace mlir;
 using namespace mlir::miopen;
 
+namespace math {
```

Comment:
as a code refactoring step, please help extract these utility routines to a separate header file.

Reply:
done